src/powershell-native/Install-PowerShellRemoting.ps1

```diff
@@ -69,9 +73,6 @@ function Register-WinRmPlugin
 
     Write-Verbose "Performing WinRM registration with: $fileName"
     reg.exe import .\$fileName
-
-    # Clean up
-#    Remove-Item .\$fileName
```

Comment:
Clean up should still take place. Just remove the "#" on line 74

Reply:
Added that line back.